admin/javascript/src/components/hidden-field/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+class HiddenFieldComponent extends SilverStripeComponent {
+
+  constructor(props) {
+    super(props);
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  render() {
+    return (
+      <div className="field hidden">
+        <input {...this.getInputProps()} />
```

Comment:
`<input {...this.props} className={['hidden', this.props.extraClass].join(' ')} type="hidden">`
or if https://github.com/JedWatson/classnames is included
`<input {...this.props} className={className(this.props.extraClass, 'hidden')} type="hidden">`